azkaban-web-server/src/main/java/azkaban/flowtrigger/quartz/FlowTriggerQuartzJob.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2017 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package azkaban.flowtrigger.quartz;
+
+import azkaban.flow.FlowUtils;
+import azkaban.flowtrigger.FlowTriggerService;
+import azkaban.project.FlowTrigger;
+import azkaban.project.Project;
+import azkaban.scheduler.AbstractQuartzJob;
+import javax.inject.Inject;
+import org.quartz.JobDataMap;
+import org.quartz.JobExecutionContext;
+
+
+public class FlowTriggerQuartzJob extends AbstractQuartzJob {
```

Comment:
I should have remembered. Does this QuartzJob run gobally only once or every flow trigger (called by user zip) will execute once?

Reply:
this job will be scheduled when project is uploaded and run based on cron schedule.